jhack/utils/simulate_event.py

```diff
@@ -6,7 +6,7 @@
 from jhack.logger import logger as jhack_logger
 
 # note juju-exec is juju-run in juju<3.0
-_J_EXEC_CMD = "juju-exec" if juju_agent_version() >= (3, 0) else "juju-run"
+_J_EXEC_CMD = "juju exec" if juju_agent_version() >= (3, 0) else "juju run"
```

Comment:
why this change? did you see errors?
Note that this command is not run on the juju cli, but in the unit. Is there a 'juju' binary in the unit?

Reply:
I reanalysed the code and indeed it works this way. Do you have any article in which you explain why it's better to run juju-exec from within the unit?